alibi_detect/od/base.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import annotations
```

Comment:
Would remove any occurrences of this as I believe it's unrelated to the functionality of the PR? There are also some unknown implications of using this now as it's integration into core Python has been deferred indefinitely.